locales/ja/wysiwyg.json

```diff
@@ -0,0 +1,326 @@
+    "Print...": "印刷...",
+    "Purple": "紫",
+    "Quotations": "引用",
+    "R": "R",
+    "Red": "赤",
+    "Redo": "やり直し",
+    "Remove color": "色の削除",
+    "Remove link": "リンクの削除",
+    "Replace": "置換",
+    "Replace all": "すべて置換",
+    "Replace with": "Replace with",
+    "Restore last draft": "最新下書きへ復旧",
+    "Rich Text Area. Press ALT-0 for help.": "リッチテキストエリア、ALT-0 ヘルプ",
+    "Rich Text Area. Press ALT-F9 for menu. Press ALT-F10 for toolbar. Press ALT-0 for help": "リッチテキストエリア、ALT-F9 メニュー、ALT-F10 ツールバー、ALT-0 ヘルプ",
+    "Right": "右",
+    "Right to left": "右 → 左",
+    "Row": "行",
+    "Row group": "行のグループ",
+    "Row properties": "行のプロパティ",
+    "Row type": "行の種類",
+    "Rows": "行",
+    "Save": "保存",
+    "Scope": "Scope",
+    "Search": "検索",
+    "Select all": "すべて選択",
+    "Select...": "選択...",
+    "Selection": "Selection",
+    "Shortcut": "ショートカット",
+    "Show caption": "キャプションの表示",
+    "Show invisible characters": "不可視文字の表示",
+    "Size": "サイズ",
+    "Source": "ソース",
+    "Source code": "ソースコード",
+    "Special character": "特殊文字",
+    "Special character...": "特殊文字...",
+    "Spellcheck": "スペルチェック",
+    "Split cell": "セルの分割",
+    "Square": "四角",
+    "Strikethrough": "取消線",
+    "Style": "スタイル",
+    "Subscript": "下付き文字",
+    "Superscript": "上付き文字",
+    "Switch to or from fullscreen mode": "全画面モードの切替",
+    "Symbols": "特殊文字",
+    "System Font": "システムフォント",
+    "Table": "テーブル",
+    "Table of Contents": "目次",
+    "Table properties": "テーブルのプロパティ",
+    "Target": "Target",
+    "Text": "テキスト",
+    "Text color": "文字色",
+    "Text to display": "表示するテキスト",
+    "The URL you entered seems to be an email address. Do you want to add the required mailto: prefix?": "入力された URL は、メールアドレスのようです。mailto: をつけますか？",
+    "The URL you entered seems to be an external link. Do you want to add the required http:// prefix?": "入力されたURLは外部リンクのようです。http:// を付けますか？",
+    "Title Case": "Title Case",
+    "To open the popup, press Shift+Enter": "To open the popup, press Shift+Enter",
+    "Tools": "ツール",
+    "Top": "上",
+    "Travel and Places": "トラベル・場所",
+    "Turquoise": "Turquoise",
+    "UPPERCASE": "UPPERCASE",
+    "Underline": "下線",
+    "Undo": "元に戻す",
+    "Update": "更新",
+    "Upload": "アップロード",
+    "Upper Alpha": "英大文字",
+    "Upper Roman": "ローマ数字大文字",
+    "Url": "URL",
+    "User Defined": "User Defined",
+    "V Align": "V Align",
+    "Valid": "Valid",
+    "Version": "Version",
+    "Vertical space": "Vertical space",
+    "View": "表示",
+    "Warn": "Warn",
+    "White": "白",
+    "Whole words": "全単語",
+    "Width": "横幅",
+    "Word count": "単語数",
+    "Words": "単語",
+    "Words: {0}": "単語: {0}",
+    "Yellow": "黄色",
+    "Yes": "はい",
+    "You have unsaved changes are you sure you want to navigate away?": "保存されていない変更点があります。削除してよろしいですか？",
+    "alignment": "alignment",
+    "comments": "コメント",
+    "example": "example",
+    "formatting": "formatting",
+    "history": "履歴",
+    "indentation": "インテント",
+    "lowercase": "lowercase",
```

Comment:
```suggestion
    "lowercase": "小文字",
```

Reply:
This corresponds to English characters, so I thought it was better not to use Japanese.

Reply:
Makes sense. Same with Uppercase.